### DISCUSSION TITLE
Feature Request: Escape Sequence for Reusing Shortcuts

### DISCUSSION THREAD

**Discussion by AlexBurdu** [nikitabobko/AeroSpace#955] [feature-ideas]
It would be great to have an escape sequence shortcut in AeroSpace that allows users to reuse a shortcut bound in AeroSpace for another application. This feature would enable users to temporarily disable AeroSpace's capture of the next shortcut, making it available for use in other apps.

**Comment by stefangstark** [comment]
You can kind of achieve this already with modes:
```
alt-x = 'mode disable'
...
[mode.disable.binding]
esc = 'mode main'
```
You'd need to remember to manually exit the disabled mode

**Comment by AlexBurdu** [comment]
Thanks, @stefangstark! That's a useful work-around.
